sdk/core/Azure.Core/src/Internal/AzureSasSignatureHistory.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Core
+{
+    /// <summary>
+    /// A marker class used to represent AzureSasCredential signature history on the <see cref="HttpMessage"/> property bag.
+    /// </summary>
+    internal class AzureSasSignatureHistory
```

Comment:
Would it make sense to add this in the policy the way we did in [RedirectPolicy](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/Pipeline/RedirectPolicy.cs#L281)?  Would it ever be used outside the policy?

Reply:
Good idea, that makes the CI changes easier for clients that rely on the new type.